creator.html

```diff
@@ -60,8 +55,8 @@ <h5 class="feature-title">No Coding!</h5>
                         </div>
                         <div class="col-10">
                             <h5 class="feature-title">Cloud Based</h5>
-                            <p class="grey-text">Kodular Creator is hosted on Google Cloud Platform. This means that your projects are safely
-                                stored on servers, so you won't have to worry about downloading them and making backups
+                            <p class="grey-text">The Creator is hosted on Google Cloud Platform. This means that your projects are safely
```

Comment:
```suggestion
                            <p class="grey-text">Kodular Creator is hosted on Google Cloud Platform. This means that your projects are safely
```

1. The name of the service is Kodular Creator, not only "Creator"